examples/advanced/vertical_xgboost/jobs/vertical_xgboost_base/app/config/config_fed_client.json

```diff
@@ -0,0 +1,76 @@
+{
+  "format_version": 2,
+  "executors": [
+    {
+      "tasks": [
+        "PSI"
+      ],
+      "executor": {
+        "id": "psi_executor",
+        "name": "PSIExecutor",
+        "args": {
+          "psi_algo_id": "dh_psi"
+        }
+      }
+    },
+    {
+      "tasks": [
+        "train"
+      ],
+      "executor": {
+        "id": "xgb_hist_executor",
+        "name": "FedXGBHistogramExecutor",
+        "args": {
+          "num_rounds": 100,
+          "early_stopping_rounds": 2,
+          "xgb_params": {
+            "validate_parameters": true,
+            "eta": 0.1,
+            "gamma": 1.0,
+            "max_depth": 8,
+            "min_child_weight": 100,
+            "tree_method": "approx",
+            "grow_policy": "depthwise",
+            "eval_metric": "auc"
+          },
+          "data_loader_id": "dataloader"
+        }
+      }
+    }
+  ],
+  "task_result_filters": [],
+  "task_data_filters": [],
+  "components": [
+    {
+      "id": "dataloader",
+      "path": "vertical_data_loader.VerticalDataLoader",
+      "args": {
+        "data_split_path": "/tmp/nvflare/vertical_xgboost/data/site-x/higgs.data.csv",
+        "label_owner": "site-1",
```

Comment:
if this is going to be substituted, we use "site-x" here as well for consistency?